shepherd/Goal.py

```diff
@@ -67,4 +78,50 @@ def apply_powerup(self, effect, alliance):
             effect - enum representing an effect to be applied to the goal
             alliance - the alliance trying to apply the effect to the goal
         """
-        pass
+        if effect == POWERUP_TYPES.TWO_X:
+            if alliance.name == CONSTANTS.BLUE:
+                if self.blue_two_x_timer.is_running():
+                    return -1
+                self.blue_two_x_timer.start_timer(CONSTANTS.TWO_X_COOLDOWN)
+                lcm_send(CONSTANTS.LCM_TARGET_SCOREBOARD,
+                         SCOREBOARD_HEADER.POWERUPS,
+                         [self.name, alliance.name, POWERUP_TYPES.TWO_X])
+                return 1
```

Comment:
Why are we returning anything? @JHGZhou

Reply:
Its mostly for feedback on bidding station. Will only need 3 results: successful powerup, incorrect code, incorrect powerup application.

Reply:
Actually in reference to this method, there should only be two return values, successful powerup or incorrect powerup application. The check of the code will happen before this method is called and this should assume the code is already correct.